content-services/content-sources-backend
# Fixes 2616: add gpg key file endpoint

## Summary
Adds an endpoint that returns the GPG key of a repository as a file.

Also adds updates the fields in the config.repo file that is returned by the config.repo file endpoint.

## Testing steps
1. Create a repository with and without a GPG Key
2. Query the `repositories/:uuid/snapshots/:snapshot_uuid/config.repo` and `repositories/:uuid/gpg_key`
3. Verify the correct returns.
